src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -8787,16 +8800,15 @@ Real64 CalcBlockenWindward(Real64 const WindAt10m,
 
     Real64 Theta; // angle between wind and surface azimuth
 
-    Theta = WindDir - SurfAzimuth - 90.0; // TODO double check theta
-    if (Theta > 180.0) Theta -= 360.0;
+    Theta = CalcWindSurfaceTheta(WindDir, SurfAzimuth);
 
     if (Theta <= 11.25) {
         Hf = 4.6 * std::pow(WindAt10m, 0.89);
-    } else if ((11.25 < Theta) && (Theta <= 33.75)) {
+    } else if (Theta <= 33.75) {
         Hf = 5.0 * std::pow(WindAt10m, 0.8);
-    } else if ((33.75 < Theta) && (Theta <= 56.25)) {
+    } else if (Theta <= 56.25) {
         Hf = 4.6 * std::pow(WindAt10m, 0.84);
-    } else if ((56.25 < Theta) && (Theta <= 100.0)) {
+    } else if (Theta <= 90.0) {
```

Comment:
A few things here:
- Don't need multiple bounding checks on theta at each condition. It should successively fall through until it hits the right one.
- This is a "windward" function, so an upper bound of 100 deg seems inappropriate. 90 deg should be the max a "windward" surface could see.

Reply:
Agreed, that makes sense. So... 91? Back to 100?